Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -629,6 +640,15 @@ void GLContext::gl_enable(GLenum capability)
     bool update_rasterizer_options = false;
 
     switch (capability) {
+    case GL_CLIP_PLANE0:
+    case GL_CLIP_PLANE1:
+    case GL_CLIP_PLANE2:
+    case GL_CLIP_PLANE3:
+    case GL_CLIP_PLANE4:
+    case GL_CLIP_PLANE5: {
+        size_t pl_idx = (size_t)capability - (size_t)GL_CLIP_PLANE0;
+        clip_plane_attrib.enabled |= (1 << pl_idx);
+    } break;
```

Comment:
Personal preference; I think it looks better to keep the `break` inside the braces.